static/app/views/performance/queues/destinationSummary/destinationSummaryPage.spec.tsx

```diff
@@ -0,0 +1,84 @@
+import {OrganizationFixture} from 'sentry-fixture/organization';
+
+import {render, screen, waitForElementToBeRemoved} from 'sentry-test/reactTestingLibrary';
+
+import {useLocation} from 'sentry/utils/useLocation';
+import useOrganization from 'sentry/utils/useOrganization';
+import usePageFilters from 'sentry/utils/usePageFilters';
+import useProjects from 'sentry/utils/useProjects';
+import DestinationSummaryPageWithProviders from 'sentry/views/performance/queues/destinationSummary/destinationSummaryPage';
+
+jest.mock('sentry/utils/useLocation');
+jest.mock('sentry/utils/usePageFilters');
+jest.mock('sentry/utils/useOrganization');
+jest.mock('sentry/utils/useProjects');
+
+describe('destinationSummaryPage', () => {
+  const organization = OrganizationFixture({features: ['performance-queues-view']});
+
+  jest.mocked(usePageFilters).mockReturnValue({
+    isReady: true,
+    desyncedFilters: new Set(),
+    pinnedFilters: new Set(),
+    shouldPersist: true,
+    selection: {
+      datetime: {
+        period: '10d',
+        start: null,
+        end: null,
+        utc: false,
+      },
+      environments: [],
+      projects: [],
+    },
+  });
+
+  jest.mocked(useLocation).mockReturnValue({
+    pathname: '',
+    search: '',
+    query: {statsPeriod: '10d', project: '1'},
+    hash: '',
+    state: undefined,
+    action: 'PUSH',
+    key: '',
+  });
+
+  jest.mocked(useOrganization).mockReturnValue(organization);
+
+  jest.mocked(useProjects).mockReturnValue({
+    projects: [],
+    onSearch: jest.fn(),
+    placeholders: [],
+    fetching: false,
+    hasMore: null,
+    fetchError: null,
+    initiallyLoaded: false,
+  });
+
+  let eventsMock, eventsStatsMock;
+
+  beforeEach(() => {
+    eventsMock = MockApiClient.addMockResponse({
+      url: `/organizations/${organization.slug}/events/`,
+      method: 'GET',
+      body: {data: []},
+    });
+
+    eventsStatsMock = MockApiClient.addMockResponse({
+      url: `/organizations/${organization.slug}/events-stats/`,
+      method: 'GET',
+      body: {data: []},
+    });
+  });
+
+  it('renders', async () => {
+    render(<DestinationSummaryPageWithProviders />);
+    await screen.findByRole('table', {name: 'Transactions'});
+    await waitForElementToBeRemoved(() => screen.queryAllByTestId('loading-indicator'));
+    screen.getByPlaceholderText('Search for events, users, tags, and more');
+    screen.getByText('Avg Latency');
+    screen.getByText('Published vs Processed');
```

Comment:
small nit: I noticed in some tests you changed these to `ByRole` queries and I think that's generally preferred over text searches. Not sure if you'd want to be consistent with that here but don't let it block this PR

Reply:
This is on the chart titles which are just `<div>` so it's just a generic role in this case, so I'm gunna stick to text for now. Probably a good idea to update the title to a header or add a label in the future